### ISSUE TITLE
long installation time due to vllm build on GPU machines

### ISSUE THREAD

**Issue by xukai92** [instructlab/instructlab#2667]
**What is the current experience of ilab you've found issue with?**
installation with GPU machine is super long (>2 hours) because of vllm building time

**How do you think the described experience should be?**
it should be fast (minutes) as building from scratch is not needed

**What is the benefit of the change you are proposing?**
users can have instructlab installed on machines with GPU much faster

**Additional context**
this the command
```
pip install vllm@git+https://github.com/opendatahub-io/vllm@v0.6.2
```

and it can be updated to
```
pip install vllm==0.6.2
```

**Comment by cdoern** [comment]
we don't use upstream vllm at the moment. I think the `vllm==0.6.2` would install `vllm-project`'s version of vllm not opendatahub.

**Comment by cdoern** [comment]
I know there are plans to make this easier and hopefully one day use proper upstream vllm!

**Comment by nathan-weinberg** [comment]
Yeah @n1hility can comment more on this but AFAIK changing vLLM versions isn't an option

**Comment by nathan-weinberg** [comment]
Closing as the vLLM version we use has changed - feel free to reopen if this is still an issue
